argoverse/evaluation/detection/utils.py

```diff
@@ -323,20 +321,24 @@ def filter_instances(
     return filtered_instances
 
 
-def rank(dts: List[ObjectLabelRecord]) -> Tuple[np.ndarray, np.ndarray]:
-    """Get the rankings for the detections, according to detector confidence.
+def rank(dts: np.ndarray) -> np.ndarray:
+    """Rank the `ObjectLabelRecord` objects in descending order by score.
 
     Args:
-        dts: Detections (N,).
+        dts: Array of `ObjectLabelRecord` objects. (N,).
 
     Returns:
-        ranks: Ranking for the detections (N,).
-        scores: Detection scores (N,).
+        ranked_dts: Array of `ObjectLabelRecord` objects ranked by score. (N,).
```

Comment:
I think the shape of the numpy should be `(MAX_NUM_BOXES, )`

Reply:
N could be << MAX_NUM_BOXES, but it would be clarify in the docstring, maybe something like:
```
ranked_dts: Array of `ObjectLabelRecord` objects ranked by score. (N,) where N <= MAX_NUM_BOXES
```

Reply:
Thanks for pointing this out, guys.